### ISSUE TITLE
Nan when using train.py

### ISSUE THREAD

**Issue by acamargosonosa** [TannerGilbert/YOLO-Tutorials#1]
I am creating a model using the tutorial of this code using 487 images, and each on them with 5 different labeled classes.  I am getting this error in the Epoch 51 when I used train.py. But, this did not happen in the first 50 Epochs, where the error was reducing in a nice way.   from 6394.2069 to  88.2426

:
:

Unfreeze all the layers:
Train on 220 samples, val on 24 samples, with batch size 32.
Epoch 51/100
6/6 [==============================] - 186s 31s/step - loss: nan - val_loss: nan
Epoch 52/100
6/6 [==============================] - 182s 30s/step - loss: nan - val_loss: nan
Epoch 53/100
...

The image size is 512x512.
I was reading on Internet that this could happen because of the loss function, regularization or learning rate. Could you please let me know what could be the problem ? Or could be because hardware issue (memory or gpu) ?

Thanks a lot,

Aldo Camargo
